### PR TITLE
WIP Fix tests in prep for moving file access in LogFactory into ConfigurationProvider 

### PR DESCRIPTION
**Pull request by ericnewton76**

### CODE REVIEW

File: tests/NLog.UnitTests/LogFactoryTests.cs [hunk 1]
```diff
@@ -48,17 +48,19 @@ public void Flush_DoNotThrowExceptionsAndTimeout_DoesNotThrow()
...
                     <targets><target type='MethodCall' name='test' methodName='Throws' className='NLog.UnitTests.LogFactoryTests, NLog.UnitTests.netfx40' /></targets>
                     <rules>
                         <logger name='*' minlevel='Debug' writeto='test'></logger>
                     </rules>
                 </nlog>");
 
-                ILogger logger = LogManager.GetCurrentClassLogger();
+                var LogFactory = new LogFactory(config);
+
+                LogFactory.ThrowExceptions = true;
```
Comment (snakefoot) [inline tests/NLog.UnitTests/LogFactoryTests.cs]: `LogManager.ThrowExceptions = true` has very special meaning when unit-testing. Why the change to `LogFactory.ThrowExceptions = true`  and enabling after the config-load?

Comment (304NotModified) [reply tests/NLog.UnitTests/LogFactoryTests.cs]: good one, I prefer also undo of this change

File: tests/NLog.UnitTests/LogFactoryTests.cs [hunk 2]
```diff
@@ -182,7 +191,7 @@ public void SecondaryLogFactoryDoesNotTakePrimaryLogFactoryLock()
             {
                 bool threadTerminated;
 
-                var primaryLogFactory = LogManager.factory;
+                var primaryLogFactory = LogManager.factory;//QUESTION should this be new LogFactory()._syncRoot test instead???
```
Comment (304NotModified) [inline tests/NLog.UnitTests/LogFactoryTests.cs]: @snakefoot do you know the answer to this question?

Comment (snakefoot) [reply tests/NLog.UnitTests/LogFactoryTests.cs]: I think it is testing that the LogFactory owned by the LogManager can live together with a secondary LogFactory-instance. And there are no calls from the secondary LogFactory into the LogManager (and the primary LogFactory-instance)

**Comment by snakefoot** [comment]
@ericnewton76 I guess you have found some other direction for your project. Maybe time to close this PR?

**Comment by 304NotModified** [comment]
Gonna close this as inactive indeed
